crates/configuration/src/configuration.rs

```diff
@@ -167,6 +172,34 @@ impl Configuration {
     }
 }
 
+#[derive(Copy, Clone, Debug, Default, Deserialize, Serialize)]
+pub struct ConfigurationOptions {
+    // Options for introspection
+    pub introspection_options: ConfigurationIntrospectionOptions,
+}
+
+#[derive(Copy, Clone, Debug, Deserialize, Serialize)]
+pub struct ConfigurationIntrospectionOptions {
```

Comment:
I hate to nitpick, but in other places where we map struct keys to config files we set `#[serde(renameAll = "camelCase")]`

Reply:
Please nitpick. Consistency is good 😄.